projects/components/src/multi-select/multi-select.component.test.ts

```diff
@@ -226,7 +228,7 @@ describe('Multi Select Component', () => {
     spectator.click(allOptionElement!);
 
     spectator.tick();
-    const selectedElements = spectator.queryAll('input:checked', { root: true });
+    const selectedElements = spectator.queryAll(CheckboxComponent);
```

Comment:
won't we still need to query from root?

Reply:
Yeah, I'd expect so - @palbizu the reason we need to query from root here is that when opened, the multiselect attaches the floating content overlay to the dom root (or more precisely, it uses popover to do so), rather than within the multiselect component (which queries are generally scoped to). Adding the extra `{ root: true }` to the queries with the new component selector would fix that, but further, the previous query on the line at least was looking not just for checkboxes, but checkboxes that were selected. To make this query equivalent, we'd end up with something like
`spectator.queryAll(lCheckboxComponent, { root: true }).filter(checkbox => checkbox.checked);`

Reply:
Resolved. thanks!